app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,253 @@
+                android:layout_width="8dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="2dp"
+                android:src="@drawable/star_icon"
+                app:layout_constraintBottom_toBottomOf="@+id/ic_rating_star_1"
+                app:layout_constraintStart_toEndOf="@id/ic_rating_star_3"
+                app:layout_constraintTop_toTopOf="@+id/ic_rating_star_1"
+                app:tint="@color/red"
+                tools:ignore="ContentDescription" />
+
+            <ImageView
+                android:id="@+id/ic_rating_star_5"
+                android:layout_width="8dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="2dp"
+                android:src="@drawable/star_icon"
+                app:layout_constraintBottom_toBottomOf="@+id/ic_rating_star_1"
+                app:layout_constraintStart_toEndOf="@id/ic_rating_star_4"
+                app:layout_constraintTop_toTopOf="@+id/ic_rating_star_1"
+                app:tint="@color/gray"
+                tools:ignore="ContentDescription" />
+
+            <TextView
+                android:id="@+id/tv_movie_list_review"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="6dp"
+                android:layout_marginBottom="8dp"
+                android:text="@string/tv_list_reviews"
+                android:textAllCaps="true"
+                android:textColor="@color/gray"
+                android:textSize="8sp"
+                android:textStyle="bold"
+                app:layout_constraintBottom_toBottomOf="@+id/iv_movies_list_image"
+                app:layout_constraintEnd_toStartOf="@id/right_fragment_guideline"
+                app:layout_constraintStart_toEndOf="@id/ic_rating_star_5" />
+
+            <TextView
+                android:id="@+id/tv_movies_list_name"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:text="@string/movie_name"
+                android:textColor="@color/white"
+                android:textSize="14sp"
+                android:textStyle="bold"
+                app:layout_constraintEnd_toEndOf="@id/right_fragment_guideline"
+                app:layout_constraintStart_toStartOf="@id/left_fragment_guideline"
+                app:layout_constraintTop_toBottomOf="@+id/iv_movies_list_image" />
+
+            <TextView
+                android:id="@+id/tv_movies_list_duration"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="8dp"
+                android:text="@string/movie_duration"
+                android:textColor="@color/gray"
+                android:textSize="8sp"
+                android:textStyle="bold"
+                app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
u have
android:layout_height="wrap_content"
in constraint where this view is nested

so
app:layout_constraintBottom_toBottomOf="parent"
looks weird

try to think in some other way